integration-tests/tests/advanced-happy-path.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('Advanced Happy path', () => {
   const componentPage = new ComponentPage();
   const latestCommitsTabPage = new LatestCommitsTabPage();
   const integrationTestsTabPage = new IntegrationTestsTabPage();
-  const sourceCodeRepoLink = 'https://github.com/devfile-samples/devfile-sample-go-basic';
+  const sourceCodeRepoLink = 'https://github.com/hac-test/devfile-sample-go-basic';
```

Comment:
Could you please remind me why is it now using our fork? Thx

Reply:
1. Added console logs to print so that we can validate it. 
2. printing env var to validate value to env is propagating to pod.
3. Also I think any future change in main repo will not impact our test as its forked.